homeassistant/components/plugwise/config_flow.py

```diff
@@ -83,6 +90,7 @@ async def async_step_zeroconf(self, discovery_info: DiscoveryInfoType):
         # pylint: disable=no-member # https://github.com/PyCQA/pylint/issues/3167
         self.context["title_placeholders"] = {
             CONF_HOST: discovery_info[CONF_HOST],
+            CONF_PORT: discovery_info[CONF_PORT],
```

Comment:
Is the port always present in the discovery information?

Reply:
It looks that way from what I gathered on info, though indeed I didn't check with all our testers (and it might be challenging to do so). I'll add it as a default to the zeroconf!

Reply:
See 4a0492a

Reply:
```suggestion
            CONF_PORT: discovery_info.get(CONF_PORT, DEFAULT_PORT),
```
In 5a0492a you add it to `self.discovery_info` but that is not what's used here, this would be better, and then the same on L109

Reply:
Check, tnx